packages/components/addon/components/hds/side-nav/list/link.hbs

```diff
@@ -0,0 +1,38 @@
+<Hds::SideNav::List::Item>
+  <Hds::Interactive
+    class="hds-side-nav__list-item-link"
+    @current-when={{@current-when}}
+    @models={{hds-link-to-models @model @models}}
+    @query={{hds-link-to-query @query}}
+    @replace={{@replace}}
+    @route={{@route}}
+    @isRouteExternal={{@isRouteExternal}}
+    @href={{@href}}
+    @isHrefExternal={{@isHrefExternal}}
+    ...attributes
```

Comment:
I think that Ember already adds an active class automatically 🤔

Reply:
yes, but this is in case in which they need to apply an "active" state programmatically (eg. it's a button, so Ember doesn't apply the `active` class; or the `current-when` doesn't work for some reasons)